### PR TITLE
Upgrade to Node 16

### PR DESCRIPTION
**Pull request by eatyourgreens**
- install and build with Node 16 Alpine.
- bump all dependencies to latest versions.
- update `server.js` with new `pg`, `yaml` and `winston` APIs.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -5,16 +5,16 @@ var fs         = require('fs');
 var SNSClient  = require('aws-snsclient');
 var winston    = require('winston');
 
-winston.add(winston.transports.File, { filename: '/app/log/ses_json.log' });
-winston.remove(winston.transports.Console);
+const logFile = new winston.transports.File({ filename: './log/ses_json.log' });
+winston.add(logFile);
```
Comment (eatyourgreens) [inline server.js:9]: The [default Winston logger](https://github.com/winstonjs/winston#using-the-default-logger) has no transports configured, so we shouldn't need to remove the console any more.

File: server.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ var fs         = require('fs');
...
 require('console-stamp')(console, '[yyyy-mm-dd HH:MM:ss.l Z]');
 
-var db_config = yaml.safeLoad(fs.readFileSync('/app/database.yml', 'utf8'));
-var auth = yaml.safeLoad(fs.readFileSync('/app/auth.yml', 'utf8'));
+var db_config = yaml.load(fs.readFileSync('./database.yml', 'utf8'));
+var auth = yaml.load(fs.readFileSync('./auth.yml', 'utf8'));
 var production_config = db_config.production;
 
-var pg_pool = pg.Pool({
+var pg_pool = new pg.Pool({
```
Comment (eatyourgreens) [inline server.js:17]: [`pg.Pool`](https://node-postgres.com/apis/pool) is a class constructor now.

File: server.js [hunk 3]
```diff
@@ -5,16 +5,16 @@ var fs         = require('fs');
...
-winston.remove(winston.transports.Console);
+const logFile = new winston.transports.File({ filename: './log/ses_json.log' });
+winston.add(logFile);
 
 require('console-stamp')(console, '[yyyy-mm-dd HH:MM:ss.l Z]');
 
-var db_config = yaml.safeLoad(fs.readFileSync('/app/database.yml', 'utf8'));
-var auth = yaml.safeLoad(fs.readFileSync('/app/auth.yml', 'utf8'));
+var db_config = yaml.load(fs.readFileSync('./database.yml', 'utf8'));
+var auth = yaml.load(fs.readFileSync('./auth.yml', 'utf8'));
```
Comment (eatyourgreens) [inline server.js:14]: [`yaml.load`](https://github.com/nodeca/js-yaml#load-string---options-) is now safe by default.

File: server.js (hunk 1 above)
Comment (yuenmichelle1) [reply server.js:9]: For some reason, when I tried to run on docker, I got an error on docker-compose up 

```
email-verify_1  | /app/node_modules/winston/lib/winston/logger.js:481
email-verify_1  |   var instance = created ? transport : (new (transport)(options));
email-verify_1  |                                         ^
email-verify_1  | 
email-verify_1  | TypeError: transport is not a constructor
```

https://github.com/winstonjs/winston/issues/1094

This ended up working. 
```suggestion
winston.add( winston.transports.File, { filename: './log/ses_json.log' });
```

Comment (eatyourgreens) [reply server.js:9]: That’s the syntax for Winston v2 but this PR upgrades to v3.

Comment (eatyourgreens) [reply server.js:9]: See the changes to `winston.Logger.add` here:
https://github.com/winstonjs/winston/blob/master/UPGRADE-3.0.md#winstonlogger

Comment (yuenmichelle1) [reply server.js:9]: ooh. i probably built from master then went to the PR to test. my bad.

**Review by yuenmichelle1 — APPROVED** [review]
looks ok to me

**Pull request merged**